testlib.h

```diff
@@ -3177,9 +3177,27 @@ void InStream::readStringTo(std::string& result)
 
     for (;;)
     {
-        cur = reader->curChar();
+        cur = reader->nextChar();
+
+        if (cur == CR)
+        {
+            cur == reader->nextChar();
```

Comment:
Should it be cur = reader->nextChar() instead of cur **==** reader->nextChar()?

Reply:
Wow that's embarassing, yes it should.